std/typecons.d

```diff
@@ -7143,3 +7143,265 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+ * Get the value of a field or property member. The member's name
+ * is resolved at runtime, rather than at compile-time.
+ *
+ * Note that currently only `pure @safe` property methods are supported.
+ *
+ * Params:
+ *     obj = the struct or class instance to access
+ *     name = the name of the field or property member
```

Comment:
Small nitpick - I usually leave the "the" here out, but I don't know what the phobos standard on this is